migtests/scripts/live-migration-fallf-run-test.sh

```diff
@@ -202,7 +202,7 @@ main() {
 	sleep 60
 
 	step "Import remaining schema (FK, index, and trigger) and Refreshing MViews if present."
-	import_schema --post-snapshot-import true --refresh-mviews true
+	finalize_schema_post_data_import --refresh-mviews true
```

Comment:
Let's not change the commands in tests for all cases, we can keep the mix of both until we support both.

Reply:
Yeah I had mostly added the change everywhere to run the command once in every situation. Will do this for now:
```
if [ $((RANDOM % 2)) -eq 0 ]; then
	finalize_schema_post_data_import --refresh-mviews=true
else
	import_schema --post-snapshot-import=true --refresh-mviews=true
fi
```
This should randomly run either of the two.

Reply:
It's probably not a big deal in this scenario but:
nit: It's generally NOT a good idea to introduce randomness into the test "scenarios". It becomes hard to reproduce if/when there is an error. Better to have separate tests that have the different scenarios.

Reply:
Hmm anyways once we deprecate the command properly, or add this to the docs we can revert back to just using the new command. For now I have removed this random thing and using either command in two of the four scripts each.